test/test_other.py

```diff
@@ -1939,6 +1939,20 @@ def test_sdl_endianness(self):
     self.run_process([EMXX, 'main.cpp'])
     self.assertContained('1234, 1234, 4321\n', self.run_js('a.out.js'))
 
+
+  def test_sdl_scan_code_from_key(self):
+    create_file('main.cpp', r'''
+      #include <stdio.h>
+      #include <SDL/SDL_keyboard.h>
+
+      int main() {
+        printf("%d\n", SDL_GetScancodeFromKey(35));
+        return 0;
+      }
+    ''')
+    self.run_process([EMXX, 'main.cpp'])
+    self.assertEqual("204\n", self.run_js('a.out.js'))
```

Comment:
I think you can just write `self.do_runf('main.cpp', '204\n')` rather than these two lines.